Code/Framework/AzCore/AzCore/Math/Geometry3DUtils.cpp

```diff
@@ -15,7 +15,7 @@ namespace AZ::Geometry3dUtils
     // Recursive method for subdividing each face of the polygon.
     namespace
     {
-        void SubdivideTriangle(
+        static void SubdivideTriangle(
```

Comment:
nit: As this is already in an anonymous namespace it is not necessary to add `static` to get internal linkage, putting a function in an unnamed namespace in C++ is equivalent so this is redundant.